src/dppm_rest_api.cr

```diff
@@ -29,7 +29,23 @@ module DppmRestApi
       false
     }
 
-    initialize_error_handlers
+    # Add authentification handler
```

Comment:
I think I prefer to keep the middleware as a separate file at least. This section will only grow over time. Is there a reason we should prefer this form of keeping fewer files?

Reply:
Better having all the code at the same place instead of multiple arbitrary small files. This file is about 50 lines, which is totally fine.

We can move this away in the future if there are good reasons, for now I don't see any.